docs/user-guide/global-configurations/catalog-framework.md

```diff
@@ -0,0 +1,85 @@
+# Catalog Framework
+
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+Ideally, a microservice/application should contain detailed information so that its users know what it does, how to use it, as well as all its technical specs. Access to such data makes it easier for engineers to quickly discover relevant microservices and also its dependent services. 
```

Comment:
Can we make this statement more generic in terms of resources instead of applications/microservices. Microservices/Clusters etc. could be portrayed as a specific example.